typescript/cli/examples/warp-route-deployment.yaml

```diff
@@ -14,6 +15,7 @@ base:
   chainName: anvil1
   type: native
   # address: 0x123... # Required for collateral types
+  # vaultAddress: 0x123... # Required for collateralVault types. Specifies the ERC4626 vault to deposit collateral into
```

Comment:
having both of these as options is somewhat misleading because the `vaultAddress` implies a collateral `address`

Reply:
Agreed. I'll modify to use `address` and remove `vaultAddress`